extensions/2.0/Vendor/EXT_property_animation/README.md

```diff
@@ -0,0 +1,111 @@
+# EXT\_property\_animation
+
+## Contributors
+
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Lewis Weaver, Microsoft
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds a specification for animation data targeting arbitrary
+properties, such as material colors, texture transform matrices and
+extension properties.
+
+## Extending Animations
+
+Property animations can be added to an animation by adding the
+`EXT_property_animation` extension to any glTF animation. For example, the
+following defines an animation with two channels that modify a material's
+baseColorFactor and roughnessFactor.
+```json
+"animations" : [
+    {
+        "channels" : [], 
+        "extensions" : {
+            "EXT_property_animation" : {
+                "channels" : [
+                    {
+                        "sampler" : 0,
+                        "target" : "/materials/1/pbrMetallicRoughness/roughnessFactor"
+                    },
+                    {
+                        "sampler" : 1,
+                        "target" : "/materials/1/pbrMetallicRoughness/baseColorFactor"
+                    }
+                ]
+            }
+        }, 
+        "samplers" : [
+            {
+                "input" : 6,
+                "interpolation" : "CUBICSPLINE",
+                "output" : 7
+            },
+            {
+                "input" : 8,
+                "interpolation" : "LINEAR",
+                "output" : 9
+            }
+        ]
+    }
+]
+ ```
+`EXT_property_animation` adds a channels list separate from the core
+specification's such that a different set of target values can be described.
+
+### sampler
+
+`sampler` has the same meaning and interpretation as the core specification's
+`sampler` property. The property animation keyframes are defined in the normal
+samplers list and the `sampler` reference indexes into this list for the
+enclosing animation.
+
+The sampler's output accessor type must match the targeted property's type.
```

Comment:
I'd propose to provide a full list of allowed accessor types for each animatable core spec property.

Reply:
Must all keyframe values comply with limitations specific to the animated property (e.g. `min`/`max` for material factors)?